python/test/unit/test_catalogues.py

```diff
@@ -15,7 +15,7 @@ def __init__(self):
         self.props = arb.neuron_cable_properties()
         try:
             self.props.catalogue = arb.load_catalogue("dummy-catalogue.so")
-        except:
+        except Exception:
```

Comment:
Seems tautological, there's no possibility of `except` catching objects not inheriting from Exception.

Reply:
https://stackoverflow.com/questions/54948548/what-is-wrong-with-using-a-bare-except

Reply:
TIL

Reply:
My pleasure